src/Functional.CQS.AOP.CommonTestInfrastructure/Functional.CQS.AOP.CommonTestInfrastructure.xml

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>Functional.CQS.AOP.CommonTestInfrastructure</name>
```

Comment:
Since this XML file is auto-generated upon builds, consider using the `.gitignore` to exclude it. You'll then have to manually remove this file from source control.

Reply:
Same goes for the other auto-generated XML files in this PR.

Reply:
Changing from absolute paths to a relative path (to `bin\Debug\...`, `bin\Release\...`) allowed me to delete these XML documents from project roots entirely